src/components/CustomerDashboard/CustomerDashboard.jsx

```diff
@@ -0,0 +1,178 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect, useState } from 'react';
+import { useHistory } from 'react-router';
+import moment from 'moment';
+import AddSample from '../AddSample/AddSample';
+
+import CustomerDetail from '../CustomerDetail/CustomerDetail';
+
+import {
+  Divider,
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Typography,
+  TextField,
+  Dialog,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 650,
+  },
+});
+
+export default function CustomerDashboard() {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const orders = useSelector((state) => state.orders);
+  const user = useSelector((state) => state.user);
+
+  const [filter, setFilter] = useState('');
+  const [openDetail, setOpenDetail] = useState(false);
+  const [clickedSample, setClickedSample] = useState({});
+
+  useEffect(() => {
+    dispatch({
+      type: 'FETCH_CUSTOMER_ORDERS',
+    });
+  }, []);
+
+  const handleOpen = (sample) => {
+    setClickedSample(sample);
+    setOpenDetail(true);
+  }; // end handleOpen
+
+  const handleClose = () => {
+    setOpenDetail(false);
+  }; // end handleClose
+
+  return (
+    <>
+      <center>
+        <Typography variant="h4" gutterBottom>
+          COMPANY ID: {user.companyID}
+        </Typography>
+        <Button
+          variant="contained"
+          style={{ backgroundColor: '#1e565c', color: 'white' }}
+          onClick={() => history.push('/addSample')}
```

Comment:
Better to use a `<Link />` component here, and style it to look like a button, for accessibility requirements.

Looks like it's may be quite easy with MaterialUI: https://stackoverflow.com/a/51642640/830030